cmake/modules/autopas_eigen.cmake

```diff
@@ -1,17 +1,41 @@
-message(STATUS "eigen3 - using bundled version")
+option(Eigen3_ForceBundled "Do not look for an installed version, always use bundled." OFF)
+
+if (NOT ${Eigen3_ForceBundled})
+    set(expectedVersion 3.3.4)
+    # capital E actually required...
+    find_package(Eigen3 ${expectedVersion} QUIET)
+    # actually I don't know our minimal supported version but this is the one I tested.
+    if (Eigen3_FOUND)
+        message(STATUS "Eigen3 - using installed system version ${Eigen3_VERSION}")
+        # to later add the alias Eigen3::Eigen needs to be promoted to global visibility
+        set_target_properties(Eigen3::Eigen PROPERTIES "IMPORTED_GLOBAL" "TRUE")
```

Comment:
this is needed because we include this cmake file in the `lib` subdirectory, correct?

Reply:
True. Targets are by default only visible in the directory they are created/imported.